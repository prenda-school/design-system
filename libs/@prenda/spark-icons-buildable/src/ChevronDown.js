import * as React from 'react';
import createSvgIcon from './utils/createSvgIcon';

export default createSvgIcon(
  <path fillRule="evenodd" d="M5.47 8.47a.75.75 0 011.06 0L12 13.94l5.47-5.47a.75.75 0 111.06 1.06l-6 6a.75.75 0 01-1.06 0l-6-6a.75.75 0 010-1.06z" />
, 'ChevronDown');
