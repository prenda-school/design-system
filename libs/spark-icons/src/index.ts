export { default as AlertCircleDuotone } from './AlertCircleDuotone';
export { default as CheckCircleDuotone } from './CheckCircleDuotone';
export { default as ChevronDown } from './ChevronDown';
export { default as HeartDuotone } from './HeartDuotone';
export { default as HomeDuotone } from './HomeDuotone';
export { default as InboxFilledDuotone } from './InboxFilledDuotone';
export { default as QuestionDuotone } from './QuestionDuotone';
export { default as RocketDuotone } from './RocketDuotone';
export { default as UserDuotone } from './UserDuotone';
export { default as UsersDuotone } from './UsersDuotone';
