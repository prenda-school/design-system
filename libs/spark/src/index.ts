export { default as Autocomplete } from './Autocomplete';
export * from './Autocomplete';

export { default as Avatar } from './Avatar';
export * from './Avatar';

export { default as Banner } from './Banner';
export * from './Banner';

export { default as Breadcrumbs } from './Breadcrumbs';
export * from './Breadcrumbs';

export { default as Box } from './Box';
export * from './Box';

export { default as Button } from './Button';
export * from './Button';

export { default as ButtonBase } from './ButtonBase';
export * from './ButtonBase';

export { default as Card } from './Card';
export * from './Card';

export { default as CardActions } from './CardActions';
export * from './CardActions';

export { default as CardContent } from './CardContent';
export * from './CardContent';

export { default as CardMedia } from './CardMedia';
export * from './CardMedia';

export { default as Checkbox } from './Checkbox';
export * from './Checkbox';

export { default as Collapse } from './Collapse';
export * from './Collapse';

export { default as CssBaseline } from './CssBaseline';

export { default as Divider } from './Divider';
export * from './Divider';

export { default as DropdownAnchor } from './DropdownAnchor';
export * from './DropdownAnchor';

export { default as DropdownContext } from './DropdownContext';
export * from './DropdownContext';

export { default as DropdownMenu } from './DropdownMenu';
export * from './DropdownMenu';

export { default as FontFacesBaseline } from './FontFacesBaseline';

export { default as FormControl } from './FormControl';
export * from './FormControl';

export { default as FormControlLabel } from './FormControlLabel';
export * from './FormControlLabel';

export { default as FormGroup } from './FormGroup';
export * from './FormGroup';

export { default as FormHelperText } from './FormHelperText';
export * from './FormHelperText';

export { default as FormLabel } from './FormLabel';
export * from './FormLabel';

export { default as Grid } from './Grid';
export * from './Grid';

export { default as IconButton } from './IconButton';
export * from './IconButton';

export { default as Input } from './Input';
export * from './Input';

export { default as InputAdornment } from './InputAdornment';
export * from './InputAdornment';

export { default as InputBase } from './InputBase';
export * from './InputBase';

export { default as InputLabel } from './InputLabel';
export * from './InputLabel';

export { default as Link } from './Link';
export * from './Link';

export { default as List } from './List';
export * from './List';

export { default as ListItem } from './ListItem';
export * from './ListItem';

export { default as ListItemAvatar } from './ListItemAvatar';
export * from './ListItemAvatar';

export { default as ListItemIcon } from './ListItemIcon';
export * from './ListItemIcon';

export { default as ListItemText } from './ListItemText';
export * from './ListItemText';

export { default as ListItemSecondaryAction } from './ListItemSecondaryAction';
export * from './ListItemSecondaryAction';

export { default as ListSubheader } from './ListSubheader';
export * from './ListSubheader';

export { default as Menu } from './Menu';
export * from './Menu';

export { default as MenuItem } from './MenuItem';
export * from './MenuItem';

export { default as MenuList } from './MenuList';
export * from './MenuList';

export { default as Pagination } from './Pagination';
export * from './Pagination';

export { default as PaginationItem } from './PaginationItem';
export * from './PaginationItem';

export { default as Paper } from './Paper';
export * from './Paper';

export { default as Radio } from './Radio';
export * from './Radio';

export { default as RadioGroup } from './RadioGroup';
export * from './RadioGroup';

export { default as ScopedCssBaseline } from './ScopedCssBaseline';
export * from './ScopedCssBaseline';

export { default as SectionMessage } from './SectionMessage';
export * from './SectionMessage';

export { default as SectionMessageTitle } from './SectionMessageTitle';
export * from './SectionMessageTitle';

export { default as Select } from './Select';
export * from './Select';

export { default as Skeleton } from './Skeleton';
export * from './Skeleton';

export { default as SparkThemeProvider } from './SparkThemeProvider';

export { default as Step } from './Step';
export * from './Step';

export { default as StepButton } from './StepButton';
export * from './StepButton';

export { default as StepConnector } from './StepConnector';
export * from './StepConnector';

export { default as StepContent } from './StepContent';
export * from './StepContent';

export { default as StepIcon } from './StepIcon';
export * from './StepIcon';

export { default as StepLabel } from './StepLabel';
export * from './StepLabel';

export { default as Stepper } from './Stepper';
export * from './Stepper';

export { default as SvgIcon } from './SvgIcon';
export * from './SvgIcon';

export { default as Switch } from './Switch';
export * from './Switch';

export { default as Tab } from './Tab';
export * from './Tab';

export { default as TabContext } from './TabContext';
export * from './TabContext';

export { default as TabList } from './TabList';
export * from './TabList';

export { default as TabPanel } from './TabPanel';
export * from './TabPanel';

export { default as Tabs } from './Tabs';
export * from './Tabs';

export { default as Tag } from './Tag';
export * from './Tag';

export { default as TextField } from './TextField';
export * from './TextField';

export { default as ThemeProvider } from './ThemeProvider';
export * from './ThemeProvider';

export { default as Toolbar } from './Toolbar';
export * from './Toolbar';

export { default as Typography } from './Typography';
export * from './Typography';

export { default as NavBar } from './NavBar';
export * from './NavBar';

export { default as NavBarButton } from './NavBarButton';
export * from './NavBarButton';

export { default as Unstable_Avatar } from './Unstable_Avatar';
export * from './Unstable_Avatar';

export { default as Unstable_AvatarButton } from './Unstable_AvatarButton';
export * from './Unstable_AvatarButton';

export { default as Unstable_Banner } from './Unstable_Banner';
export * from './Unstable_Banner';

export { default as Unstable_Button } from './Unstable_Button';
export * from './Unstable_Button';

export { default as Unstable_Checkbox } from './Unstable_Checkbox';
export * from './Unstable_Checkbox';

export { default as Unstable_CheckboxField } from './Unstable_CheckboxField';
export * from './Unstable_CheckboxField';

export { default as Unstable_CheckboxListItem } from './Unstable_CheckboxListItem';
export * from './Unstable_CheckboxListItem';

export { default as Unstable_CheckboxMenuItem } from './Unstable_CheckboxMenuItem';
export * from './Unstable_CheckboxMenuItem';

export { default as Unstable_CheckboxGroupField } from './Unstable_CheckboxGroupField';
export * from './Unstable_CheckboxGroupField';

export { default as Unstable_ContentGroup } from './Unstable_ContentGroup';
export * from './Unstable_ContentGroup';

export { default as Unstable_CssBaseline } from './Unstable_CssBaseline';

export { default as Unstable_Divider } from './Unstable_Divider';
export * from './Unstable_Divider';

export { default as Unstable_FontFacesBaseline } from './Unstable_FontFacesBaseline';

export { default as Unstable_FormControl } from './Unstable_FormControl';
export * from './Unstable_FormControl';

export { default as Unstable_FormControlLabel } from './Unstable_FormControlLabel';
export * from './Unstable_FormControlLabel';

export { default as Unstable_FormGroup } from './Unstable_FormGroup';
export * from './Unstable_FormGroup';

export { default as Unstable_FormHelperText } from './Unstable_FormHelperText';
export * from './Unstable_FormHelperText';

export { default as Unstable_FormLabel } from './Unstable_FormLabel';
export * from './Unstable_FormLabel';

export { default as Unstable_IconButton } from './Unstable_IconButton';
export * from './Unstable_IconButton';

export { default as Unstable_Input } from './Unstable_Input';
export * from './Unstable_Input';

export { default as Unstable_InputAdornment } from './Unstable_InputAdornment';
export * from './Unstable_InputAdornment';

export { default as Unstable_Link } from './Unstable_Link';
export * from './Unstable_Link';

export { default as Unstable_List } from './Unstable_List';
export * from './Unstable_List';

export { default as Unstable_ListItem } from './Unstable_ListItem';
export * from './Unstable_ListItem';

export { default as Unstable_ListSubheader } from './Unstable_ListSubheader';
export * from './Unstable_ListSubheader';

export { default as Unstable_Menu } from './Unstable_Menu';
export * from './Unstable_Menu';

export { default as Unstable_MenuItem } from './Unstable_MenuItem';
export * from './Unstable_MenuItem';

export { default as Unstable_MenuList } from './Unstable_MenuList';
export * from './Unstable_MenuList';

export { default as Unstable_ModalDialog } from './Unstable_ModalDialog';
export * from './Unstable_ModalDialog';

export { default as Unstable_ModalDialogActions } from './Unstable_ModalDialogActions';
export * from './Unstable_ModalDialogActions';

export { default as Unstable_ModalDialogContent } from './Unstable_ModalDialogContent';
export * from './Unstable_ModalDialogContent';

export { default as Unstable_ModalDialogContentText } from './Unstable_ModalDialogContentText';
export * from './Unstable_ModalDialogContentText';

export { default as Unstable_ModalDialogTitle } from './Unstable_ModalDialogTitle';
export * from './Unstable_ModalDialogTitle';

export { default as Unstable_Paper } from './Unstable_Paper';
export * from './Unstable_Paper';

export { default as Unstable_Radio } from './Unstable_Radio';
export * from './Unstable_Radio';

export { default as Unstable_RadioField } from './Unstable_RadioField';
export * from './Unstable_RadioField';

export { default as Unstable_RadioGroup } from './Unstable_RadioGroup';
export * from './Unstable_RadioGroup';

export { default as Unstable_RadioGroupField } from './Unstable_RadioGroupField';
export * from './Unstable_RadioGroupField';

export { default as Unstable_SectionMessage } from './Unstable_SectionMessage';
export * from './Unstable_SectionMessage';

export { default as Unstable_Select } from './Unstable_Select';
export * from './Unstable_Select';

export { default as Unstable_SvgIcon } from './Unstable_SvgIcon';
export * from './Unstable_SvgIcon';

export { default as Unstable_Switch } from './Unstable_Switch';
export * from './Unstable_Switch';

export { default as Unstable_SwitchField } from './Unstable_SwitchField';
export * from './Unstable_SwitchField';

export { default as Unstable_Tag } from './Unstable_Tag';
export * from './Unstable_Tag';

export { default as Unstable_Tooltip } from './Unstable_Tooltip';
export * from './Unstable_Tooltip';

export { default as Unstable_TextField } from './Unstable_TextField';
export * from './Unstable_TextField';

export { default as Unstable_Typography } from './Unstable_Typography';
export * from './Unstable_Typography';

export { default as makeStyles } from './makeStyles';

export { default as styled } from './styled';
export * from './styled';

export { default as theme } from './theme';
export * from './theme';

export { default as unstable_createSvgIcon } from './unstable_createSvgIcon';

export { default as useMediaQuery } from './useMediaQuery';

export { default as useTheme } from './useTheme';

export { default as withStyles } from './withStyles';
export * from './withStyles';
