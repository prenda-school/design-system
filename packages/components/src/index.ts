export * from './Arc';
export * from './ArcCircle';
export * from './ArcSegments';
export * from './ArcSegmentsSweep';
export * from './ArcSweep';
export * from './ArcUnits';
export * from './ArcUnitLabel';
export * from './ArcScale';
export * from './Bar';
export * from './BarFill';
export * from './BarScale';
export * from './BarUnits';
export * from './BarUnitLabel';
export * from './Chart';
