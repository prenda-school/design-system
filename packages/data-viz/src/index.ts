export * from './lib/data-viz';
