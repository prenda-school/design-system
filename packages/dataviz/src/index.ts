export * from './RadialBar';
export * from './RadialBarCircle';
export * from './RadialBarScale';
export * from './RadialBarSegments';
export * from './RadialBarSegmentsSweep';
export * from './RadialBarSweep';
export * from './RadialBarUnitLabel';
export * from './RadialBarUnits';
