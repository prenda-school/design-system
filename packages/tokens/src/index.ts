export { backgroundColor } from './background-color';
export type {
  BackgroundColorToken,
  BackgroundColorPalette,
} from './background-color';

export { border } from './border';
export type {
  BorderToken,
  BorderPalette,
  BorderNestedPalettes,
} from './border';

export { boxShadow } from './box-shadow';
export type {
  BoxShadowToken,
  BoxShadowPalette,
  BoxShadowNestedPalettes,
} from './box-shadow';

export { color } from './color';
export type { ColorToken, ColorPalette, ColorNestedPalettes } from './color';

export * from './font-family';
export * from './font-feature-settings';
export * from './font-size';

export { radius } from './radius';
export type { RadiusToken, RadiusPalette } from './radius';

export { skinTone } from './skin-tone';
export type {
  SkinToneToken,
  SkinTonePalette,
  SkinToneNestedPalettes,
} from './skin-tone';

export { textColor } from './text-color';
export type { TextColorToken, TextColorPalette } from './text-color';
