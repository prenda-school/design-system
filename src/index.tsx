export { Button, ButtonProps } from './actions/';
export {
  Menu,
  MenuProps,
  MenuItemOptions,
  Checkbox,
  CheckboxProps,
  Input,
  InputProps,
  Radio,
  RadioProps,
} from './form-inputs/';
export {
  SparkThemeProvider,
  TertiaryColor,
  muiTheme,
  prendaTheme,
} from './theme/';
export {
  BookIllustration,
  CollaborateIllustration,
  ConquerIllustration,
  CreateIllustration,
} from './illustrations/';
export {
  PrendaWordmarkDark,
  PrendaWordmarkLight,
  PrendaMonogramDark,
  PrendaMonogramLight,
  SparkLogoDark,
  SparkLogoLight
} from './logos';
export {
  CaretUpIcon,
  CaretDownIcon,
  CaretLeftIcon,
  CaretRightIcon,
  PointerIcon,
  SearchIcon,
  CheckIcon,
  CheckCircleIcon,
  CheckCircleFilledIcon,
  XIcon,
  XCircleIcon,
  XCircleFilledIcon,
  AlertIcon,
  AlertFilledIcon,
  HeartIcon,
  HeartFilledIcon,
  PlayIcon,
  MenuIcon,
  DesktopIcon,
  EnvelopeIcon,
  PencilIcon,
  PinIcon,
  CalendarIcon,
  SchoolIcon,
  CaretAltUpIcon,
  CaretAltDownIcon,
  CaretAltLeftIcon,
  CaretAltRightIcon,
  TrendUpIcon,
  TrendDownIcon,
  PlusIcon,
  PlusCircleIcon,
  PlusCircleFilledIcon,
  MinusIcon,
  MinusCircleIcon,
  MinusCircleFilledIcon,
  BoxIcon,
  CheckboxIcon,
  CheckboxFilledIcon,
  QuestionCircleIcon,
  QuestionCircleFilledIcon,
  SortIcon,
  BoltIcon,
  BoltFilledIcon,
  HomeIcon,
  UserIcon,
  RefreshIcon,
  ClockIcon,
  StopwatchIcon,
  BookIcon,
  CodeIcon,
  StarIcon,
  StarFilledIcon,
} from './icons';
export { Typography, TypographyProps } from './typography';