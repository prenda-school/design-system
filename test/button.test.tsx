import { Button } from '../src/';
import initStoryshots from '@storybook/addon-storyshots';
initStoryshots();

describe('Button', () => {
  it('is truthy', () => {
    expect(Button).toBeTruthy();
  });
});
