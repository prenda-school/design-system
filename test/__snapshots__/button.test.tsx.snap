// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots prenda-spark/Button Configurable Button 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeMedium"
  disabled={false}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Disabled Label Only Large 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeLarge Mui-disabled"
  disabled={true}
  labelOnly={true}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={-1}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Disabled Label Only Medium 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeMedium Mui-disabled"
  disabled={true}
  labelOnly={true}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={-1}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Disabled Label Only Small 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeSmall Mui-disabled"
  disabled={true}
  labelOnly={true}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={-1}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Disabled Large 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeLarge Mui-disabled"
  disabled={true}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={-1}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Disabled Medium 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeMedium Mui-disabled"
  disabled={true}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={-1}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Disabled Outlined Large 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeLarge Mui-disabled"
  disabled={true}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  outlined={true}
  tabIndex={-1}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Disabled Outlined Medium 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeMedium Mui-disabled"
  disabled={true}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  outlined={true}
  tabIndex={-1}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Disabled Outlined Small 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeSmall Mui-disabled"
  disabled={true}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  outlined={true}
  tabIndex={-1}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Disabled Small 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeSmall Mui-disabled"
  disabled={true}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={-1}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Large 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeLarge"
  disabled={false}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Large Icon 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeLarge"
  disabled={false}
  icon={<Memo />}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="SparkButton-content"
  />
</button>
`;

exports[`Storyshots prenda-spark/Button Large Label Only 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeLarge"
  disabled={false}
  labelOnly={true}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Medium 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeMedium"
  disabled={false}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Medium Icon 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeMedium"
  disabled={false}
  icon={<Memo />}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="SparkButton-content"
  />
</button>
`;

exports[`Storyshots prenda-spark/Button Medium Label Only 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeMedium"
  disabled={false}
  labelOnly={true}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Outlined Large 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeLarge"
  disabled={false}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  outlined={true}
  tabIndex={0}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Outlined Medium 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeMedium"
  disabled={false}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  outlined={true}
  tabIndex={0}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Outlined Small 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeSmall"
  disabled={false}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  outlined={true}
  tabIndex={0}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Small 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeSmall"
  disabled={false}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;

exports[`Storyshots prenda-spark/Button Small Icon 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeSmall"
  disabled={false}
  icon={<Memo />}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="SparkButton-content"
  />
</button>
`;

exports[`Storyshots prenda-spark/Button Small Label Only 1`] = `
<button
  className="MuiButtonBase-root SparkButton-root SparkButton-variantSolid SparkButton-sizeSmall"
  disabled={false}
  labelOnly={true}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="SparkButton-content"
  >
    PizzaTaco
  </span>
</button>
`;
